WebRoot/css/login.css

```diff
@@ -107,23 +237,56 @@ FORM {
 	position:relative;
 	z-index:20;
 }
+
 .LoginScreen #ZLoginWhatsThisAnchor {
-	color:@LoginLabelColor@;
-	font-size:.9em;
-	margin-left:5px;
+	color: #1e282ec2;
+	font-size: .9em;
+	margin-left: 8px;
+	position: absolute;
+	left: 100%;
+	TOP: 8px;
+	outline: none;
+}
+
+#dialogCloseButton {
+	float: right;
+	width: 10;
+	height: 10;
+	margin: -6px -18px 0px 0px;
 }
+
 .LoginScreen #ZLoginForgotPassword {
-	color:@LoginLabelColor@;
-	font-size:.9em;
+	color: #59a3dd;
+	font-size: .9em;
 	float: right;
+	padding: 2px;
+	outline: none;
 }
+.passwordWrapper {
+	position: relative;
+}
+
+.passwordWrapper #showAndHide{
+	position: absolute;
+	z-index: 2;
+	font-size: 12px;
+	color: #828282;
+	right: 10px;
+	top: 11px;
+	cursor: pointer;
+}
+
 .LoginScreen #ZLoginWhatsThis {
-	left:0px;
-	margin-left:-10em;
-	position:absolute;
-	top:25px;
-	width:40em;
-	z-index:30; 
+	margin: 0em 1em 0 -2em;
+	position: absolute;
+	top: 25px;
+	z-index: 3;
+	height: 350px;
+	font-size: 14px;
+	padding: 24px 32px;
+	border-radius: 5px;
+	border-style: none;
+	box-shadow: 0 4px 6px 0 rgba(0,0,0,.4);
```

Comment:
need to re think about this.

Reply:
how about this 

position: absolute;
    top: 23px;
    z-index: 3;
    height: 350px;
    font-size: 14px;
    left: 15px;
    right: 15px;
    padding: 24px 32px;
    border: 5px none;
    box-shadow: 0 4px 6px 0 rgba(0,0,0,.4);

Reply:
how about this 

```
position: absolute;
    top: 23px;
    z-index: 3;
    height: 350px;
    font-size: 14px;
    left: 15px;
    right: 15px;
    padding: 24px 32px;
    border: 5px none;
    box-shadow: 0 4px 6px 0 rgba(0,0,0,.4);
```